bundles/org.openhab.ui/web/src/components/widgets/standard/oh-gauge-card.vue

```diff
@@ -3,8 +3,8 @@
     <f7-card-header v-if="config.title">
       <div>{{ config.title }}</div>
     </f7-card-header>
-    <f7-card-content class="oh-gauge-card display-flex justify-content-center">
-      <oh-gauge :context="childContext(context.component)" @command="onCommand" />
+    <f7-card-content class="oh-gauge-card display-flex justify-content-center clickPointer" @click.native="performAction" >
+      <oh-gauge :context="childContext(context.component)" @command="onCommand"  />
     </f7-card-content>
```

Comment:
I would prefer the following:
```suggestion
    <f7-card-content class="oh-gauge-card display-flex justify-content-center">
      <f7-link v-if="config.action" class="oh-gauge-link" @click="performAction">
        <oh-gauge :context="childContext(context.component)" />
      </f7-link>
      <oh-gauge v-else :context="childContext(context.component)" />
    </f7-card-content>
```
That way, you get the hand pointer as well as a visual feedback when you hit the control - but only if an action is defined.

See also below for the definition of the `oh-gauge-link` CSS class.